src/test-run/index.js

```diff
@@ -261,3 +300,42 @@ ServiceMessages[CLIENT_MESSAGES.ready] = function (msg) {
 
     return new Promise((resolve, reject) => this.pendingRequest = { resolve, reject, responseTimeout });
 };
+
+ServiceMessages[CLIENT_MESSAGES.executePendingBrowserManipulation] = async function (msg) {
+    this.debugLog.driverMessage(msg);
+
+    var command        = this.pendingWindowManipulation;
+    var executionError = null;
+
+    if (command.type === COMMAND_TYPE.takeScreenshot) {
+        try {
+            await this.screenshotCapturer.captureAction({
+                pageUrl:    msg.pageUrl,
+                stepName:   uuid.v4(),
+                customPath: command.customPath
+            });
+        }
+        catch (err) {
+            // NOTE: swallow the error silently if we can't take screenshots for some
+            // reason (e.g. we don't have permissions to write a screenshot file).
+            executionError = null;
```

Comment:
I added it for consistent https://github.com/DevExpress/testcafe/pull/543/files#diff-945208629fe0c4eea6d7e13aaeb4ca5bR44